packages/netsuite-adapter/src/change_validators/unique_fields.ts

```diff
@@ -29,21 +29,21 @@ const { mapValuesAsync } = promises.object
 
 const FIELD_DEFAULT_NAME = 'FIELD_DEFAULT_NAME'
 
-type RestrictedType = 'savedSearch' | 'financialLayout' | 'customRecordField'
+type RestrictedType = 'savedSearch' | 'financialLayout' | 'customRecordField' | 'workflow' | 'script'
 
 type GetterParams = {
   elemID: ElemID
   elementsSource: ReadOnlyElementsSource
 }
 
 type ElementGetter = (params: GetterParams) => Promise<string[]>
-type ChangeGetter = (change: ChangeDataType) => string
+type ChangeGetter = (change: ChangeDataType) => string[]
 
 type RestrictedTypeGetters = {
   getChangeRestrictedField: ChangeGetter
```

Comment:
This should be changed to plural, right?

```suggestion
  getChangeRestrictedFields: ChangeGetter
```